### DISCUSSION TITLE
Language selector: Is there a way to switch to the same page in the other language?

### DISCUSSION THREAD

**Discussion by syeda-git** [squidfunk/mkdocs-material#3507] [Q&A] [answered]
Hi,

I have set up a basic language selector site https://syeda-git.github.io/mkdocs-sandbox/en/mydoc_about/
Currently, if i switch from that page to French, the root(index) page in FR is displayed.

The expectation is that when i switch from a page, the same page in the target language should be displayed.
So, if i switch from https://syeda-git.github.io/mkdocs-sandbox/en/mydoc_about/ to **FR**, i should see
https://syeda-git.github.io/mkdocs-sandbox/fr/mydoc_about/

Is that possible by default or some Javascripting is required?

Thanks...
Syed

**Comment by squidfunk** [comment, marked as answer]
It's currently not supported. However, it's already on the [roadmap](https://squidfunk.github.io/mkdocs-material/insiders/#8000-scotch-bonnet). I can't say when I'll find the time to investigate it, but it will be definitely added at some point. In the meantime, you might be successful using JavaScript, or the [`mkdocs-static-i18n`](https://github.com/ultrabug/mkdocs-static-i18n) plugin which I understand has basic support for this feature.

**Reply by syeda-git** [reply]
Thanks for the quick response.
